axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -1099,6 +1099,95 @@
           "address": "0x9186233C327329408a9034F22407244421cADbEF"
         }
       }
+    },
+    "hyperliquid": {
+      "name": "Hyperliquid",
+      "axelarId": "hyperliquid",
+      "networkType": "testnet",
+      "chainId": 998,
+      "rpc": "https://rpc.hyperliquid-testnet.xyz/evm",
+      "tokenSymbol": "HYPE",
+      "confirmations": 1,
+      "finality": "finalized",
+      "decimals": 18,
+      "approxFinalityWaitTime": 1,
+      "chainType": "evm",
+      "explorer": {
+        "name": "Hyperliquid-testnet Explorer",
+        "url": "https://app.hyperliquid-testnet.xyz/explorer"
+      },
+      "contracts": {
+        "AxelarGateway": {
+          "deployer": "0x640fF4a6D70AD92fc064633586D0A51FA8D89Ba5",
```

Comment:
which key is this? Should use the shared team key for devnet deployments

Reply:
the scripts should also print a warning that the predicted address doesn't match before you deploy to catch issues